lib/gstreamer/msdk/transcoder.py

```diff
@@ -15,16 +15,19 @@
 
 @slash.requires(*have_gst_element("msdk"))
 @slash.requires(using_compatible_driver)
+@slash.requires(*have_gst_element("va"))
```

Comment:
We should not have this here.  If user disables gst-va plugin build and just enables gst-msdk plugin build, then they cannot test gst-msdk cases only because this `slash.requires` would fail and cause ALL transcode cases to skip.

The `have_gst_element("va...")` in the `requirements` dict (below) is enough to have va_hw cases automatically skipped in that scenario.

Reply:
Understood. I removed this line, thanks